src/types/types.ts

```diff
@@ -60,6 +60,12 @@ export type Question = {
 export type QuestionComponentProps = {
   question: Question;
   setAnswer: (question: Question, input: any) => void;
+  existingAnswer?: any;
+};
+
+export type QuestionManagerProps = {
+  setNextScreen: (...args: string[]) => void;
```

Comment:
Curious - how is this setting the next screen?

Reply:
it sets the next screen in the cycle, so like it tells the formsscreen which screen should come next: currently it is

general -> additionalScreen
additionalScreen -> general

so even though it is going backwards (addiitional -> general) it is still the next screen if that makes sense, idk i can refactor this later if necessary